### PR TITLE
Add support for hand tracking

### PR DESCRIPTION
**Pull request by grozdanov**
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved terminology
- [ ] You have added unit tests

1. Implement setter function for property `Prism.trackHandGesture` in the PrismBuilder.
2. Set `MxsBaseApp.eventListener` to return false in order to forward the server events to the PrismController(s).

### CODE REVIEW

**Comment by grozdanov** [comment]
Hand tracking application requirements:
1. The application needs to be immersive !
2. Required privileges for tracking hand gestures:
- `GesturesConfig`
- `GesturesSubscribe`
3. Sample code:
```
import React from 'react';
import { Prism, Scene, Text, Content,  } from 'magic-script-components';

const handGestures = [
  'hand-c',
  'hand-dynamic-closing-hand',
  'hand-dynamic-horizontal-wag',
  'hand-dynamic-thumbs-down',
  'hand-dynamic-thumbs-up',
  'hand-dynamic-vertical-wag',
  'hand-finger',
  'hand-fist',
  'hand-l',
  'hand-ok',
  'hand-open-hand-back',
  'hand-pinch',
  'hand-thumb',
  'hand-tracking-gesture-down',
  'hand-tracking-gesture-up'
];

export default class MyApp extends React.Component {
  constructor (props) {
    super(props);

    this.state = {
      message: 'Waiting for hand gesture.'
    };
  }

  onGestureInputEventHandler = (eventData) => {
    const gestureType = eventData.Gesture;
    console.log(`Gesture: ${gestureType}`);
    this.setState({message: gestureType});
    return true;
  };

  onEventHandler = (eventData) => {
    if (eventData.constructor.name === 'GestureInputEventData') {
      console.log('GestureInputEvent: ', eventData.Gesture);
    }
    return true;
  };

  render () {
    return (
      <Scene>
        <Prism
          size={[3.0, 3.0, 3.0]}
          position={[0, 0, -1]}
          positionRelativeToCamera={true}
          trackHandGesture={handGestures}
          selected={true}
        >
          <Content
            name="rootNode"
            onGestureInputEvent={this.onGestureInputEventHandler}
            // onEvent={this.onEventHandler}
          >
            <Text textSize={0.05}>
              {this.state.message}
            </Text>
          </Content>
        </Prism>
      </Scene>
    );
  }
}
```

File: src/platform/lumin-runtime/mxs-immersive-app.js [hunk 1]
```diff
@@ -45,8 +45,14 @@ export class MxsImmersiveApp extends ImmersiveApp {
...
-    return this._baseApp.addPrism(properties, this);
-  }
+    const prism = this._baseApp.addPrism(properties, this);
+    const selected = properties.selected;
+
+    if (selected !== undefined && typeof selected === 'boolean') {
+      // ImmersiveApp only API !
+      this.selectPrism(prism, selected);
+    }
+    return prism;  }
```
Comment (kpiascik) [inline src/platform/lumin-runtime/mxs-immersive-app.js:55]: the closing `}` should be on a new line

File: src/platform/lumin-runtime/mxs-base-app.js [hunk 2]
```diff
@@ -82,15 +82,18 @@ export class MxsBaseApp {
...
-
   addPrism(properties, app) {
     const prismSize = properties.size;
-
     if (!Array.isArray(prismSize)) {
       throw new TypeError(`Prism size is not a vec3: ${prismSize}`);
     }
 
-    const prism = executor.callNativeFunction(app, 'requestNewPrism', prismSize);
+    const prismType = properties.type !== undefined ? properties.type : 'world';
```
Comment (kpiascik) [inline src/platform/lumin-runtime/mxs-base-app.js:91]: We should call out in the documentation that `World` is the default.

Comment (grozdanov) [reply src/platform/lumin-runtime/mxs-base-app.js:91]: Agree ! The docs need to be updated.

File: src/platform/lumin-runtime/mxs-immersive-app.js (hunk 1 above)
Comment (grozdanov) [reply src/platform/lumin-runtime/mxs-immersive-app.js:55]: 🤦

**Pull request merged**